### PR TITLE
LPS-178520 Update POSHI automation | Master

### PR DESCRIPTION
**Pull request by alessandromalves**
This is only for master
https://issues.liferay.com/browse/LPS-178520
cc.: @david-gutierrez-mesa @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by alessandromalves** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/utilitypage/UtilityPagesWithStaging.testcase [hunk 1]
```diff
@@ -127,4 +127,50 @@ definition {
...
+		}
+
+		task ("Then the previously created page should remain marked as default") {
+			PagesAdmin.openPagesAdmin(siteURLKey = "test-site-name");
+
+			Navigator.gotoNavTab(navTab = "Utility Pages");
+
+			LexiconCard.viewCardSticker(
+				card = "404 Error Utility Page",
+				icon = "check-circle");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/utilitypage/UtilityPagesWithStaging.testcase:172]: Only need to add this step into UtilityPagesWithStaging#Apply404ErrorUtilityPage
After line 97 and line120

**Review by Tim-Cao — COMMENTED** [review]
Hi @alessandromalves, would you mind refactoring according to comments? Thanks

**Comment by alessandromalves** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
LGTM! @alessandromalves

**Comment by Tim-Cao** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/137451

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**